### PR TITLE
Initial Configurations

### PR DESCRIPTION
**Pull request by Avarjana**
## Description
Basic configurations for Logstash, Kibana and WSO2 IS is added with this pull request.

### CODE REVIEW

File: Kibana/kibana-7-10.ndjson [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{"exportedCount":24,"missingRefCount":0,"missingReferences":[]}
```
Comment (inthirakumaaran) [inline Kibana/kibana-7-10.ndjson]: add a new line to the bottom

Comment (inthirakumaaran) [reply Kibana/kibana-7-10.ndjson]: check other places as well

**Review by inthirakumaaran — APPROVED** [review]

File: WSO2-IS-Configs/deployment.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+[server]
+hostname = "localhost"
+node_ip = "127.0.0.1"
+base_path = "https://$ref{server.hostname}:${carbon.management.port}"
+
+[super_admin]
+username = "admin"
+password = "admin"
```
Comment (omindu) [inline WSO2-IS-Configs/deployment.toml]: mask passwords in all places

**Comment by Avarjana** [comment]
Please review and merge @inthirakumaaran

**Review by omindu — APPROVED** [review]

File: Kibana/Exports/kibana-7-10.ndjson [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+{"exportedCount":24,"missingRefCount":0,"missingReferences":[]}
```
Comment (bhagyasakalanka) [inline Kibana/Exports/kibana-7-10.ndjson:25]: Shall we add a newline. Look for other places as well

File: Elasticsearch/elasticsearch.yml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+#
+# Use a descriptive name for your cluster:
+#
+#cluster.name: my-application
+#
+# ------------------------------------ Node ------------------------------------
+#
+# Use a descriptive name for the node:
+#
+#node.name: node-1
```
Comment (lashinijay) [inline Elasticsearch/elasticsearch.yml:23]: formatting issue, space is missing after `#`. check other places as well.

File: Logstash/logstash_filebeats.conf [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+		source => "eventdata"
+		remove_field => ["eventdata","message"]
+	}
+	#geo ip filter to transform the IP address to location
+	geoip {
+    		source => "[event][payloadData][remoteIp]"
+    	}
+
+	#ruby code to convert the comma seperated roles into an array
+	ruby{
```
Comment (lashinijay) [inline Logstash/logstash_filebeats.conf:24]: formatting issue

File: Logstash/logstash_filebeats.conf [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+			index => "auth"
+		}
+	} else if [eventType] == "session" {
+		elasticsearch {
+			hosts => ["localhost:9200"]
+			index => "session"
+			document_id => "%{[event][payloadData][sessionId]}"
+		}
+	}
+
```
Comment (lashinijay) [inline Logstash/logstash_filebeats.conf:71]: Let's remove the unnecessary new line here.

File: Logstash/logstash_http.conf [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+			index => "auth"
+		}
+	} else if [headers][eventtype] == "session"{
+		elasticsearch {
+			hosts => ["localhost:9200"]
+			index => "session"
+			document_id => "%{[event][payloadData][sessionId]}"
+		}
+	}
+
```
Comment (lashinijay) [inline Logstash/logstash_http.conf:68]: Let's remove the redundant new line here. check the other places also.

File: Logstash/logstash_logger.conf [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+	  drop {}
+	}
+	grok {
+		match => {"message" => " Unique ID: %{GREEDYDATA:eventType},\n Event: %{GREEDYDATA:eventdata}"}
+	}
+	json {
+		source => "eventdata"
+		remove_field => ["eventdata","message"]
+	}
+	#geo ip filter to transform the IP address to location
```
Comment (lashinijay) [inline Logstash/logstash_logger.conf:23]: Let's format the comments properly. Check other places also.

File: Postman Collection/WSO2 IS Analytics ELK.postman_collection.json [hunk 9]
```diff
@@ -0,0 +1,486 @@
+{
+	"info": {
+		"_postman_id": "00c15e99-73cb-4b0e-be98-a0d5f09858ba",
+		"name": "WSO2 IS Analytics",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Login attempts per day view (calendar)",
```
Comment (lashinijay) [inline Postman Collection/WSO2 IS Analytics ELK.postman_collection.json:9]: indentation issue. let's fix in other places also.

File: Postman Collection/WSO2 IS Analytics ELK.postman_collection.json [hunk 10]
```diff
@@ -0,0 +1,486 @@
...
+						"session",
+						"_search"
+					]
+				}
+			},
+			"response": []
+		}
+	],
+	"protocolProfileBehavior": {}
+}
```
Comment (lashinijay) [inline Postman Collection/WSO2 IS Analytics ELK.postman_collection.json:486]: Let's add a new line here.

File: Kibana/Exports/kibana-7-10.ndjson (hunk 3 above)
Comment (Avarjana) [reply Kibana/Exports/kibana-7-10.ndjson:25]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14

File: Elasticsearch/elasticsearch.yml (hunk 4 above)
Comment (Avarjana) [reply Elasticsearch/elasticsearch.yml:23]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/15

File: Logstash/logstash_filebeats.conf (hunk 5 above)
Comment (Avarjana) [reply Logstash/logstash_filebeats.conf:24]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/15

File: Logstash/logstash_filebeats.conf (hunk 6 above)
Comment (Avarjana) [reply Logstash/logstash_filebeats.conf:71]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/15

File: Logstash/logstash_http.conf (hunk 7 above)
Comment (Avarjana) [reply Logstash/logstash_http.conf:68]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/15

File: Logstash/logstash_logger.conf (hunk 8 above)
Comment (Avarjana) [reply Logstash/logstash_logger.conf:23]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/15

File: Postman Collection/WSO2 IS Analytics ELK.postman_collection.json (hunk 9 above)
Comment (Avarjana) [reply Postman Collection/WSO2 IS Analytics ELK.postman_collection.json:9]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/15

File: Postman Collection/WSO2 IS Analytics ELK.postman_collection.json (hunk 10 above)
Comment (Avarjana) [reply Postman Collection/WSO2 IS Analytics ELK.postman_collection.json:486]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/15

**Pull request merged**